apps/web/src/modules/job/list/components/FilterContainer/index.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  ArrowDropDownOutlined,
+  DateRangeOutlined,
+  Schedule,
+} from '@mui/icons-material'
+import {
+  Checkbox,
+  Divider,
+  FormControlLabel,
+  FormGroup,
+  Typography,
+} from '@mui/material'
+import React from 'react'
+
+import TextField from '../../../../../common/components/TextField'
+import { FilterBox } from './style'
+
+export default function SearchBox() {
+  return (
+    <FilterBox>
+      <Typography variant="body1"> การถ่ายทำ </Typography>
+      <TextField
+        fullWidth
+        label="วันถ่ายทำ"
+        InputProps={{ endAdornment: <DateRangeOutlined color="primary" /> }}
+      />
+      <TextField
+        fullWidth
+        label="เวลาเริ่มต้นการถ่ายทำ"
+        InputProps={{ endAdornment: <Schedule color="primary" /> }}
+      />
+      <TextField
+        fullWidth
+        label="เวลาสิ้นสุดการถ่ายทำ"
+        InputProps={{ endAdornment: <Schedule color="primary" /> }}
+      />
```

Comment:
This quite weird that we need to manual fill (maybe see this https://mui.com/x/react-date-pickers/time-picker/)